plugins/woocommerce/src/Internal/Traits/OrderAttributionMeta.php

```diff
@@ -230,19 +230,39 @@ private function get_hpos_order_object( $post_or_order ) {
 		return $post_or_order;
 	}
 
+
 	/**
-	 * Map posted values to meta values.
+	 * Map posted, prefixed values to fields.
+	 * Used for the classic forms.
 	 *
-	 * @param array $raw_values The raw values from the submitted data.
+	 * @param array $raw_values The raw values from the POST form.
 	 *
 	 * @return array
 	 */
-	private function get_source_values( array $raw_values = array() ): array {
+	private function get_unprefixed_fields( array $raw_values = array() ): array {
 		$values = array();
 
 		// Look through each field in POST data.
 		foreach ( $this->fields as $field ) {
-			$value = sanitize_text_field( wp_unslash( $raw_values[ $this->get_prefixed_field( $field ) ] ?? '(none)' ) );
+			$values[ $field ] = $raw_values[ $this->get_prefixed_field( $field ) ];
```

Comment:
As you mentioned in discussion, this line needs the `??` null coalescing operator that got moved into `get_source_values` because `$_POST` may not include all these values

Reply:
Thanks, fixed in https://github.com/woocommerce/woocommerce/commit/cda715786ad23b83712ddd126c5d85903fbc7332